syntax="proto3";

package astre.file;

import "File/proto/world_chunk.proto";

message SaveArchiveData
{
    repeated WorldChunk chunks = 1;
};